packages/sd-jwt/src/action-handler.ts

```diff
@@ -175,22 +187,34 @@ export class SDJwtPlugin implements IAgentPlugin {
   async verify(sdjwt: SDJwtVcInstance, context: IRequiredContext, data: string, signature: string) {
     const decodedVC = await sdjwt.decode(`${data}.${signature}`)
```

Comment:
Why are you involving the KMS on the verify function? The verify happens 99% of the time on some-one elses SD-JWT. And even if it were our own, we still should not involve the KMS for verification. The KMS is only involved in issuing/signing/presenting

Reply:
yeah. correct. I have removed the kms calls from this part of the code which is responsible for verifying and added functions for getting the `JWK` from certificate chain of the issuer did. this way we're not calling into our KMS for fetching a public key